gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/services/reflect/UpdateLoader.java

```diff
@@ -323,8 +323,8 @@ InputStream getResourceAsStream(String name) {
 			return is;
 
 		// match behaviour of standard class loaders. 
-		if (name.endsWith(".class"))
-			return null;
+/*		if (name.endsWith(".class"))
```

Comment:
add a comment here with the issue number to clarify why this has been removed

Reply:
Ok. Will add it.